Modules/Loadable/Markups/VTKWidgets/vtkSlicerMarkupsWidgetRepresentation3D.cxx

```diff
@@ -1206,9 +1205,10 @@ void vtkSlicerMarkupsWidgetRepresentation3D::UpdateViewScaleFactor()
   {
     return;
   }
-  if (!this->Renderer->GetRenderWindow()->GetGenericContext())
+
+  if (this->Renderer->GetRenderWindow()->GetNeverRendered())
```

Comment:
@LucasGandel @sankhesh  The change implemented by @Sunderlandkyl but since the function `GetNeverRendered` is only used a few times in VTK itself, I was wondering if this was an anti-pattern of if you had a better idea for handling this.

Reply:
IMO, it is fine to query the `NeverRendered` flag. There are a few widgets that use that flag too.